### PR TITLE
Improve setup tutorial docs

### PR DESCRIPTION
**Pull request by pnasrat**
Update installation instructions to use editable install of deployer

Add section on install pre-commit hooks

### CODE REVIEW

File: docs/tutorials/setup.md [hunk 1]
```diff
@@ -86,13 +86,24 @@ libraries we will use. Again this depends on how you installed python, and wheth
...
+Now you are all ready to use our deployer scripts! Note if dependencies get
+added you will to run the installation again.
 
 Remember you need to *activate the environment* you installed these libraries into each
 time you use any of our scripts.
 
-## Step 3: Authenticate with Google Cloud to decrypt our secret files
+## Step 3: Setup the git pre-commit hooks
+
+In the root of the checked out `infrastructure` repo, with your choice of Python virtual
```
Comment (yuvipanda) [inline docs/tutorials/setup.md:99]: `pre-commit` doesn't *require* to be installed with your virtual environment activated, and I personally think installing it outside of your virtual environment (via `brew` or `apt` or similar) is better. `pre-commit` also creates its own sub-virtual environments to install the configured pre-commit handlers and not touch the current env.

So I would suggest here just linking out to the installation instructions page and say 'install it'?

Comment (pnasrat) [reply docs/tutorials/setup.md:99]: fair enough. I just assumed hermetic environment for everything in the repo as I didn't dive into how pre-commit works

File: docs/tutorials/setup.md [hunk 2]
```diff
@@ -86,13 +86,26 @@ libraries we will use. Again this depends on how you installed python, and wheth
...
 
 Remember you need to *activate the environment* you installed these libraries into each
 time you use any of our scripts.
 
-## Step 3: Authenticate with Google Cloud to decrypt our secret files
+## Step 3: Setup the git pre-commit hooks
+
+Install pre-commit [pre-commit installation instruction](https://pre-commit.com/#introduction)
+
+In the root of the checked out `infrastructure` repo run the below ton install
```
Comment (yuvipanda) [inline docs/tutorials/setup.md:101]:
```suggestion
In the root of the checked out `infrastructure` repo run the below to install
```

**Review by yuvipanda — APPROVED** [review]
One typo otherwise lgtm.

**Review by consideRatio — APPROVED** [review]

**Pull request merged**